.github/workflows/agw-coverage.yml

```diff
@@ -110,22 +110,24 @@ jobs:
         run: |
           echo "Available storage:"
           df -h
-      - name: Notify Bazel C/C++ coverage failure to slack
+      # Slack notification configuration
+      - name: "Set Slack channel #ci as default"
+        run: echo "webhook=${{ secrets.SLACK_WEBHOOK }}" >> $GITHUB_ENV
+      - name: "Set Slack channel #ci-info for failures"
```

Comment:
Why would we want a separate channel for failures different from default?

Reply:
Because failures might have another priority than just messages (and in turn than success messages).
As comparison: also bash has a `stdout` and `stderr` channel. So I think this makes a lot of sense.